samples/subsys/ipc/ipc_service/static_vrings/src/main.c

```diff
@@ -231,14 +242,32 @@ static void ipc1_entry(void *dummy0, void *dummy1, void *dummy2)
 	k_sem_take(&ipc1_bound_sem, K_FOREVER);
 
 	while (message < 100) {
-		ret = ipc_service_send(&ipc1_ept, &message, sizeof(message));
+		uint32_t len = sizeof(message);
```

Comment:
I think it would be good to use `uint32_t len = 0;` in one communication direction, to verify both requesting buffer of given size, or maximal buffer size.

Reply:
Could I just leave this as is? If I force push I'll lose the two ACKs and I have to beg people for those again.

Reply:
Sure, the sample can be improved later